include/hw/i386/virt.h

```diff
@@ -46,7 +47,7 @@ typedef struct {
     /* GSI */
     qemu_irq *gsi;
 
-    PCIBus *pci_bus;
+    PCIBus *pci_bus[MAX_PCI_SEGMENT];
```

Comment:
We should make this a list vs an array. That will give us more flexibility and the ability to add to this list.